<!DOCTYPE html>
<html lang="en">

  <!-- Head -->
  <head>    <!-- Metadata, OpenGraph and Schema.org -->
    

    <!-- Standard metadata -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <title>Bardia  Yousefi | NSCLC</title>
    <meta name="author" content="Bardia  Yousefi" />
    <meta name="description" content="None-small cell lung cancer" />
    <meta name="keywords" content="Bardia Yousefi, faculty, CV, publications, Data-mining, Medical image analysis, Medical data analysis, pattern recognition, machine learning, Deep Learning." />


    <!-- Bootstrap & MDB -->
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous" />

    <!-- Fonts & Icons -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous">
    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">

    <!-- Code Syntax Highlighting -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="none" id="highlight_theme_light" />

    <!-- Styles -->
    
    <link rel="shortcut icon" href="/assets/img/LOGO.png"/>
    
    <link rel="stylesheet" href="/assets/css/main.css">
    <link rel="canonical" href="http://localhost:4000/projects/1_project/">
    
    <!-- Dark Mode -->
    
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark" />

    <script src="/assets/js/theme.js"></script>
    <script src="/assets/js/dark_mode.js"></script>
    

  </head>

  <!-- Body -->
  <body class="fixed-top-nav ">

    <!-- Header -->
    <header>

      <!-- Nav Bar -->
      <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
        <div class="container">
          <a class="navbar-brand title font-weight-lighter" href="http://localhost:4000/"><span class="font-weight-bold">Bardia</span>   Yousefi</a>
          <!-- Navbar Toggle -->
          <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar top-bar"></span>
            <span class="icon-bar middle-bar"></span>
            <span class="icon-bar bottom-bar"></span>
          </button>

          <div class="collapse navbar-collapse text-right" id="navbarNav">
            <ul class="navbar-nav ml-auto flex-nowrap">

              <!-- About -->
              <li class="nav-item ">
                <a class="nav-link" href="/">about</a>
              </li>
              

              <!-- Other pages -->
              <li class="nav-item ">
                <a class="nav-link" href="/publications/">publications</a>
              </li>
              <li class="nav-item ">
                <a class="nav-link" href="/Research/">research</a>
              </li>
              <li class="nav-item ">
                <a class="nav-link" href="/teaching/">teaching</a>
              </li>
              <li class="nav-item dropdown ">
                <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">weblinks</a>
                <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown">
                  <a class="dropdown-item" href="https://biocomp.umd.edu/" target="_blank" rel="noopener noreferrer">BCE</a>
                  <a class="dropdown-item" href="https://bioe.umd.edu/" target="_blank" rel="noopener noreferrer">BIOE</a>
                  <a class="dropdown-item" href="https://eng.umd.edu/" target="_blank" rel="noopener noreferrer">James Clark Eng.</a>
                  <a class="dropdown-item" href="http://vision.gel.ulaval.ca/en/index.php/" target="_blank" rel="noopener noreferrer">CVSL</a>
                  <a class="dropdown-item" href="https://www.gelgif.ulaval.ca/accueil/" target="_blank" rel="noopener noreferrer">ECE-ULaval</a>
                  <a class="dropdown-item" href="http://www2.ulaval.ca/" target="_blank" rel="noopener noreferrer">ULaval</a>
                  <a class="dropdown-item" href="http://mivim.gel.ulaval.ca/?Lang=1" target="_blank" rel="noopener noreferrer">MIVIM</a>
                </div>
              </li>

              <!-- Toogle theme mode -->
              <div class="toggle-container">
                <a id="light-toggle">
                  <i class="fas fa-moon"></i>
                  <i class="fas fa-sun"></i>
                </a>
              </div>
            </ul>
          </div>
        </div>
      </nav>
    </header>

    <!-- Content -->
    <div class="container mt-5">
      <!-- page.html -->
        <div class="post">

          <header class="post-header">
            <h1 class="post-title">NSCLC</h1>
            <p class="post-description">None-small cell lung cancer</p>
          </header>

          <article>
            <p><strong>Liquid biopsy in NSCLC</strong></p>

<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>---
Key Points:
•	Two radiomic phenotypes for EGFR-mutated advanced NSCLC were identified which
showed significant separation of K-M curves for PFS (p = 0.03) but not for OS.  

•	When predicting PFS, Cox regression modeling using clinical covariates and number
of mutations yielded a c-statistic of 0.73; when radiomic phenotype was added to the
model the c-statistic increased to 0.77, with LRT p = 0.01 compared to the model
without phenotype, and HR = 3.80 [95%CI (1.35, 10.69)] for phenotype 2 versus 1. The
PFS augmented model showed separation of K-M curves (p&lt;0.001) when split at the
median prognostic score.

•	When predicting OS, Cox regression modeling using clinical covariates and number
of mutations yielded a c-statistic of 0.80; when radiomic phenotype was added to the
model the c-statistic increased to 0.83, with LRT p = 0.08 compared to the model
without phenotype, and HR = 3.88 [95%CI (0.73, 20.5)] for phenotype 2 versus 1. The
OS augmented model showed separation of K-M curves (p=0.003) when split at the
median prognostic score.
---
</code></pre></div></div>

<p>The discovery of activating mutations and the development of targeted therapies has improved survival in patients with non-small cell lung cancer (NSCLC) (1). Mutation detection by tissue and circulating tumor DNA (ctDNA) next-generation sequencing (NGS) guides therapy selection both at initial diagnosis and disease progression. Epidermal growth factor receptor (EGFR) mutations are the most common therapeutically targetable variants in NSCLC, and treatment with an EGFR tyrosine kinase inhibitor (TKI) has shown superior efficacy compared to standard chemotherapy in mutation-positive patients (2). However, primary resistance occurs in 20-30% of patients (3). Ultimately, all patients develop acquired resistance to EGFR-directed therapies and an active area of research is the use of novel combination therapies, including antibodies against c-met, poly-adenosine diphosphate ribose polymerase inhibitors and antiangiogenic therapies along with EGFR-TKIs to improve long-term efficacy (4, 5).</p>

<div class="row">
    <div class="col-sm mt-3 mt-md-0">
        <figure>

  <picture>
    <source media="(max-width: 480px)" srcset="/assets/img/EGFR_0-480.webp"></source>
    <source media="(max-width: 800px)" srcset="/assets/img/EGFR_0-800.webp"></source>
    <source media="(max-width: 1400px)" srcset="/assets/img/EGFR_0-1400.webp"></source>
    <!-- Fallback to the original file -->
    <img class="img-fluid rounded z-depth-1" src="/assets/img/EGFR_0.png" title="example image">

  </picture>

</figure>

    </div>
</div>
<div class="caption">
    Tumor segmentation and radiomic analysis. a) Example of segmentation of a tumor expressing the epidermal growth factor receptor (EGFR) T790M mutation. b) Workflow of radiomics analysis where the tumor is segmented in 3D, followed by radiomic feature extraction, and two-level hierarchical clustering to first reduce feature dimensionality and then cluster the derived radiomic signatures into distinct tumor phenotypes.
</div>

<p>Tumor heterogeneity is thought to play a role in TKI response and is associated with poor outcome (6-9), as EGFR mutations may be suboptimal targets when they co-occur with genetic alternations or are subclonally expressed (8, 9). Small tissue biopsies may not fully reflect tumor heterogeneity and can often be difficult to obtain (10, 11), with tissue NGS only able to be completed for as few as 50% of patients (12). Thus, developing non-invasive tests to assess the likelihood of response to an EGFR-TKI is critical for therapy selection. Studies have shown that ctDNA analysis represents a non-invasive biomarker that can improve targetable mutation detection, and that ctDNA molecular heterogeneity predicts clinical outcome (13-15). Although useful clinically, however, ctDNA sensitivity remains less than ideal (13).</p>

<p>An emerging non-invasive approach to characterize tumor heterogeneity is to analyze tumor imaging phenotypes (16, 17). Radiomics analysis enables the detection of tumor imaging features and patterns of intra-tumor heterogeneity not appreciable by the human eye, increasing the wealth of information from radiological imaging. Studies specifically suggest that radiomic analysis may provide novel prognostic markers related to gene-expression patterns and responder signatures for NSCLC patients receiving targeted therapy (18-31). Most studies to date have focused on using radiomic analysis on computed tomography (CT) and/or positron emission tomography (PET)/CT data to predict EGFR mutation status, using statistical modeling or machine learning approaches for reducing the high dimensionality of radiomic features (19, 21-29, 32). More recently deep learning approaches have also been used to predict outcomes after TKI therapy for NSCLC (31, 33). While this field is rapidly developing, a question still remains as to which extent radiomic analysis can complement established prognostic markers for TKIs, as most studies have either evaluated radiomic features in the absence of established prognostic biomarkers or have only examined surrogate endpoints, such as EGFR mutation status, rather than actual patient outcomes. In addition, and to the best of our knowledge, no studies have evaluated radiomic analysis in the context of complementing liquid biopsy-based assessment, which is another promising non-invasive tool for characterizing tumor heterogeneity when predicting EGFR-TKIs response.</p>

<div class="row justify-content-sm-center">
    <div class="col-sm-8 mt-3 mt-md-0">
        <figure>

  <picture>
    <source media="(max-width: 480px)" srcset="/assets/img/EGFR_2-480.webp"></source>
    <source media="(max-width: 800px)" srcset="/assets/img/EGFR_2-800.webp"></source>
    <source media="(max-width: 1400px)" srcset="/assets/img/EGFR_2-1400.webp"></source>
    <!-- Fallback to the original file -->
    <img class="img-fluid rounded z-depth-1" src="/assets/img/EGFR_2.png" title="example image">

  </picture>

</figure>

    </div>
</div>
<div class="caption">
    3D tumor volume. 3D tumor volumes for four segmentation cases and two different NSCLC Radiogenomics datasets.
</div>

<p>The purpose of our study was to determine the feasibility of integrating radiomics features with ctDNA next-generation sequencing data to predict TKI outcomes in EGFR mutant NSCLC. Our approach combines unsupervised hierarchical clustering and principal component analysis (PCA) of radiomic features extracted from clinically acquired CT scans, to arrive at two distinct radiomic phenotypes. Our hypothesis is that integrating these radiomic phenotypes with ctDNA and clinical variables can improve assessment of tumor heterogeneity and outcome prediction to EGFR-targeted therapy for metastatic NSCLC.</p>

<div class="row justify-content-sm-center">
    <div class="col-sm-8 mt-3 mt-md-0">
        <figure>

  <picture>
    <source media="(max-width: 480px)" srcset="/assets/img/EGFR_1-480.webp"></source>
    <source media="(max-width: 800px)" srcset="/assets/img/EGFR_1-800.webp"></source>
    <source media="(max-width: 1400px)" srcset="/assets/img/EGFR_1-1400.webp"></source>
    <!-- Fallback to the original file -->
    <img class="img-fluid rounded z-depth-1" src="/assets/img/EGFR_1.png" title="example image">

  </picture>

</figure>

    </div>
</div>
<div class="caption">
    Tumors cluster in the two phenotypes. Visualizations of the original CT images with tumors in field of view for phenotypes 1 (n=21) and 2 (n=19). Most cancers in phenotype 1 appear to be relatively smaller, with elongated shape, convex borders and adjacent linear opacities, while cancers in phenotype 2 are generally larger and have more ground-glass, irregular, and indistinct border characteristics suggestive of potential inflammatory changes that may be related to their observed worse PFS and OS outcomes. The tumor area is highlighted by 5-10% opacity for demonstration purposes.
</div>

<p>Summary statement: Our results suggest that the combination of three non-invasively obtained characteristics, circulating tumor DNA (ctDNA), radiomic features from standard of care CT, and clinical variables, can help improve prediction of response to EGFR-targeted therapy. This may enhance management and therapy selection for patients with non-small cell lung cancer.</p>

<div class="row justify-content-sm-center">
    <div class="col-sm-8 mt-3 mt-md-0">
        <figure>

  <picture>
    <source media="(max-width: 480px)" srcset="/assets/img/EGFR_3-480.webp"></source>
    <source media="(max-width: 800px)" srcset="/assets/img/EGFR_3-800.webp"></source>
    <source media="(max-width: 1400px)" srcset="/assets/img/EGFR_3-1400.webp"></source>
    <!-- Fallback to the original file -->
    <img class="img-fluid rounded z-depth-1" src="/assets/img/EGFR_3.png" title="example image">

  </picture>

</figure>

    </div>
</div>
<div class="caption">
    Survival analysis by line of therapy. Kaplan-Meier curves for (top row) progression-free survival and (bottom row) overall survival in first-line patients (left) and second- and third-line patients (right), showing that the radiomic tumor phenotypes can further sub-stratify patients in the second or third line of treatment.
</div>

<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>---
Combining radiomic phenotypes of non-small cell lung cancer with liquid biopsy data may improve prediction of response to EGFR inhibitors
B Yousefi, MJ LaRiviere, EA Cohen, TH Buckingham, SS Yee, TA Black, ...
Scientific reports 11 (1), 1-13
---
</code></pre></div></div>

<hr>

<p><strong>Interobserver Variability</strong></p>

<div class="row">
    <div class="col-sm mt-3 mt-md-0">
        <figure>

  <picture>
    <source media="(max-width: 480px)" srcset="/assets/img/hA04-480.webp"></source>
    <source media="(max-width: 800px)" srcset="/assets/img/hA04-800.webp"></source>
    <source media="(max-width: 1400px)" srcset="/assets/img/hA04-1400.webp"></source>
    <!-- Fallback to the original file -->
    <img class="img-fluid rounded z-depth-1" src="/assets/img/hA04.png" title="example image">

  </picture>

</figure>

    </div>
    <div class="col-sm mt-3 mt-md-0">
        <figure>

  <picture>
    <source media="(max-width: 480px)" srcset="/assets/img/hC04-480.webp"></source>
    <source media="(max-width: 800px)" srcset="/assets/img/hC04-800.webp"></source>
    <source media="(max-width: 1400px)" srcset="/assets/img/hC04-1400.webp"></source>
    <!-- Fallback to the original file -->
    <img class="img-fluid rounded z-depth-1" src="/assets/img/hC04.png" title="example image">

  </picture>

</figure>

    </div>
    <div class="col-sm mt-3 mt-md-0">
        <figure>

  <picture>
    <source media="(max-width: 480px)" srcset="/assets/img/h3D04-480.webp"></source>
    <source media="(max-width: 800px)" srcset="/assets/img/h3D04-800.webp"></source>
    <source media="(max-width: 1400px)" srcset="/assets/img/h3D04-1400.webp"></source>
    <!-- Fallback to the original file -->
    <img class="img-fluid rounded z-depth-1" src="/assets/img/h3D04.png" title="example image">

  </picture>

</figure>

    </div>
</div>

<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>---
This study tackles interobserver variability with respect to specialty training in manual
segmentation of non-small cell lung cancer (NSCLC). Four readers included for segmentation
are: a data scientist (BY), a medical student (LS), a radiology trainee (MH), and a specialty
trained radiologist (SK) for a total of 295 patients from two publicly available databases.
Sørensen–Dice (SD) coefficients and low rank Pearson correlation coefficients (CC) of
429 radiomics were calculated to assess interobserver variability. Cox proportional
hazard (CPH) models and Kaplan-Meier (KM) curves of overall survival (OS)
prediction for each dataset were also generated.
---
</code></pre></div></div>

<p>Lung cancer is the leading cause of cancer-related death in the United States. Non-small cell lung cancer (NSCLC) represents the majority of primary lung cancers and carries a poor prognosis and low overall survival. Computed tomography (CT) is a routinely used diagnostic imaging tool in clinical management in oncology due to the ability of CT to noninvasively provide anatomic information for detection, staging, and therapy response assessment. Over the past decade it has become evident that quantitative features are embedded in conventional medical imaging data, not appreciable to the human eye. These radiomics features are a reflection of tissue architecture, heterogeneity, and pericellular environment and can be harnessed to construct tissue signatures that correlate with clinically relevant biomarkers, including tumor histologic subtype, mutational status, degree of infiltration with tumor infiltrating lymphocytes, as well as therapeutic endpoints such as overall survival. These imaging “phenotypes” provide valuable data that may enhance personalization of medical care in oncology.
It is well known that repeatability and reproducibility of radiomic features on CT are sensitive to various image details such as image acquisition settings, processing, reconstruction algorithm, and specific software used for radiomic feature extraction. Furthermore, certain radiomic features are more sensitive to these variations than others, with first order features, specifically entropy, consistently reported as being very stable while other texture features, such as coarseness and contrast, being the least reproducible.</p>

<div class="row">
    <div class="col-sm mt-3 mt-md-0">
        <figure>

  <picture>
    <source media="(max-width: 480px)" srcset="/assets/img/IOV_1-480.webp"></source>
    <source media="(max-width: 800px)" srcset="/assets/img/IOV_1-800.webp"></source>
    <source media="(max-width: 1400px)" srcset="/assets/img/IOV_1-1400.webp"></source>
    <!-- Fallback to the original file -->
    <img class="img-fluid rounded z-depth-1" src="/assets/img/IOV_1.png" title="example image">

  </picture>

</figure>

    </div>
</div>
<div class="caption">
    Workflow of the approach. The NSCLC tumor is segmented from the original CT images by four segmenters (n = 4) with different backgrounds, yielding radiomics features and tumor masks as inputs. Next, PCA categorizes features based on their maximum variance in radiomics. For every group, three principal components of feature sets are selected and used for correlative analysis and prediction of survival.
</div>

<p>Discovery of predictive and prognostic radiomic features in cancer is currently of great interest to the radiologic community; however, there is no reliable fully automated means of segmenting lung cancer. Tumor delineation and contouring are often performed by scientists with a range of training in anatomical imaging including imaging analysts, students, physician trainees, and attending physicians using either manual or semi-automated techniques. In addition to being time consuming, 3-dimensional manual and semi-automated contouring are subject to interobserver variability. This variability has been shown to be particularly challenging with segmented lesions when associated with ground glass components and post-obstructive atelectasis. In order to generate high fidelity phenotypic radiomic signatures, tumor segmentations must be reproducible across different readers. Performing quality segmentations is an important task. Although the ability to anticipate tumor histology, mutational status, and therapeutic consequences are all ultimate goals of radiomics, interobserver variability between readers should be thoroughly investigated before subsequent feature analysis is tested, given that these segmentations form the basis of the analyses.</p>

<div class="row justify-content-sm-center">
    <div class="col-sm-8 mt-3 mt-md-0">
        <figure>

  <picture>
    <source media="(max-width: 480px)" srcset="/assets/img/IOV_2-480.webp"></source>
    <source media="(max-width: 800px)" srcset="/assets/img/IOV_2-800.webp"></source>
    <source media="(max-width: 1400px)" srcset="/assets/img/IOV_2-1400.webp"></source>
    <!-- Fallback to the original file -->
    <img class="img-fluid rounded z-depth-1" src="/assets/img/IOV_2.png" title="example image">

  </picture>

</figure>

    </div>
</div>
<div class="caption">
    3D tumor volume. 3D tumor volumes for four segmentation cases and two different NSCLC Radiogenomics datasets.
</div>

<p>Discovery of predictive and prognostic radiomic features in cancer is currently of great interest to the radiologic and oncologic community. Tumor phenotypic and prognostic information can be obtained by extracting features on tumor segmentations, and it is typically imaging analysts, physician trainees, and attending physicians who provide these labeled datasets for analysis. The potential impact of level and type of specialty training on interobserver variability in manual segmentation of NSCLC was examined. Although there was some variability in segmentation between readers, the subsequently extracted radiomic features were overall well correlated. High fidelity radiomic feature extraction relies on accurate feature extraction from imaging that produce robust prognostic and predictive radiomic NSCLC biomarkers. This study concludes that this goal can be obtained using segmenters of different levels of training and clinical experience.</p>

<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>---
Impact of Interobserver Variability in Manual Segmentation of Non-Small Cell Lung Cancer (NSCLC) Applying Low-Rank Radiomic Representation on Computed Tomography
M Hershman, B Yousefi, L Serletti, M Galperin-Aizenberg, L Roshkovan, ...
Cancers 13 (23), 5985
---
</code></pre></div></div>

          </article>

        </div>

    </div>

    <!-- Footer -->    
    <footer class="fixed-bottom">
      <div class="container mt-0">
        © Copyright 2022 Bardia  Yousefi. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="noopener noreferrer">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" target="_blank" rel="noopener noreferrer">al-folio</a> theme.

      </div>
    </footer>

    <!-- JavaScripts -->
    <!-- jQuery -->
  <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>

    <!-- Bootsrap & MDB scripts -->
  <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.2/dist/umd/popper.min.js" integrity="sha256-l/1pMF/+J4TThfgARS6KwWrk/egwuVvhRzfLAMQ6Ds4=" crossorigin="anonymous"></script>
  <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.min.js" integrity="sha256-SyTu6CwrfOhaznYZPoolVw2rxoY7lKYKQvqbtqN93HI=" crossorigin="anonymous"></script>
  <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script>

    <!-- Masonry & imagesLoaded -->
  <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script>
  <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
  <script defer src="/assets/js/masonry.js" type="text/javascript"></script>
    
  <!-- Medium Zoom JS -->
  <script src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.6/dist/medium-zoom.min.js" integrity="sha256-EdPgYcPk/IIrw7FYeuJQexva49pVRZNmt3LculEr7zM=" crossorigin="anonymous"></script>
  <script src="/assets/js/zoom.js"></script><!-- Load Common JS -->
  <script src="/assets/js/common.js"></script>

    <!-- MathJax -->
  <script type="text/javascript">
    window.MathJax = {
      tex: {
        tags: 'ams'
      }
    };
  </script>
  <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script>
  <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>

    
  </body>
</html>

